configure.ac

```diff
@@ -82,6 +82,9 @@ LIBNOTIFY_REQUIRED=0.5.1
 GCR_REQUIRED=3.5.5
 AVAHI_REQUIRED=0.6.22
 GVDB_REQUIRED="glib-2.0 >= $GLIB_REQUIRED"
+JSON_GLIB_REQUIRED=1.2.0
+NETTLE_REQUIRED=3.2
+HOGWEED_REQUIRED=3.2
```

Comment:
Be sure to alphabetize these when you rebase. (You'll have a merge conflict here; they're alphabetized in master.)